website/docs/podman/accessing-podman-from-another-wsl-instance.md

```diff
@@ -0,0 +1,235 @@
+---
+sidebar_position: 60
+title: Accessing Podman from another WSL distribution
+description: On Windows, access your Podman Desktop containers from another Windows Subsystem for Linux (WSL) distribution.
+---
+
+# Accessing Podman from another WSL distribution
+
+On Windows, [Podman Desktop creates a Windows Subsystem for Linux (WSL) virtual machine: the Podman Machine](/docs/podman/creating-a-podman-machine.md).
+It also configures the Windows Podman client to communicate with the Podman Machine.
+However, it does not configure your other WSL distributions.
+
+You might have other WSL distributions running, and want to access from there to your Podman Desktop containers.
+
+This tutorial focuses on the most common context to walk you through the steps to configure your WSL distribution:
+
+- Ubuntu distribution of Linux.
+- Default Podman Machine.
+
+In foldable details, you can find alternative steps for least common contexts:
+
+- Custom WSL distribution.
+- Custom Podman Machine.
+
+## Configuring your WSL distribution
+
+1. Start a session in your WSL distribution:
+
+   ```shell-session
+   > wsl
+   ```
+
+1. To communicate with the remote Podman Machine, you need a Podman client.
+
+   To benefit from the latest features, such as `podman kube` subcommands, and to enable the remote mode by default, consider installing the `podman-remote` binary rather than the `podman` package from the distribution.
+
+   Download the `podman-remote` binary from the [Podman releases page](https://github.com/containers/podman/releases/latest).
+
+1. Configure the Podman client in your WSL distribution to communicate with the remote Podman machine defined by Podman Desktop.
+
+   This will ensure consistency when you are working with Podman from all your different environments
+
+   Set the default Podman system connection to your Podman Machine:
+
+   ```shell-session
+   $ podman system connection add --default podman-machine-default-root unix:///mnt/wsl/podman-sockets/podman-machine-default/podman-root.sock
+   ```
+
+   <details>
+   <summary>
+   On a custom Podman Machine, the remote Podman Machine destination might be different.
+
```

Comment:
@MarkCheshire Would it address your concern with the fact that the default story is for a rootful Podman?

```suggestion

Two parameters can change:

- The machine name might differ from `podman-machine-default`.
- The socket name, for a rootless Podman machine with root privileges disabled.

```